src/vm/crossbitness.h

```diff
@@ -0,0 +1,46 @@
+#ifndef CROSSBITNESS_H
+#define CROSSBITNESS_H
+
+struct CrossBitness
+{
+#ifdef _TARGET_64BIT_
+    typedef INT64 OBJECTREF;
+#else
+    typedef INT32 OBJECTREF;
+#endif
+
+    struct ObjHeader
+    {
+#ifdef _TARGET_64BIT_
+        DWORD    m_alignpad;
+#endif // _TARGET_64BIT_
+        Volatile<DWORD> m_SyncBlockValue;
+    };
+
+    struct Object
+    {
+        DWORD m_pMethTab;
+    };
+
+    struct ArrayBase : public Object
+    {
+        DWORD m_NumComponents;
+#ifdef _TARGET_64BIT_
+        DWORD pad;
+#endif // _TARGET_64BIT_
+    };
+
+    struct StringObject : public Object
```

Comment:
I do not see this used.

Reply:
I guess it was needed before https://github.com/dotnet/coreclr/pull/17876/

Reply:
Yes, you're right. It was needed before for ObjSizeOf(StringObject) in StringObject::GetSize.